runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -515,10 +641,22 @@ TR_VectorAPIExpansion::visitNodeToBuildVectorAliases(TR::Node *node)
                if (getReturnType(methodSymbol) ==  Scalar) continue; // OK to use by any other parent node
                }
 
-            if (_trace)
-               traceMsg(comp(), "Invalidating #%d since it's used by unsupported node %p\n",
-                                 child->getSymbolReference()->getReferenceNumber(), node);
-             invalidateSymRef(child->getSymbolReference());
+            if (boxingAllowed() &&
+                child->getOpCode().isFunctionCall())
+               {
+               if (_trace)
+                   traceMsg(comp(), "Invalidating10 node %p since it's a child of unsupported node %p (%s)\n",
+                                     child, node, node->getOpCode().getName());
+               invalidateNode(child);
+               }
+            else
+               {
+               if (_trace)
+                   traceMsg(comp(), "Invalidating10 #%d since it's used by unsupported node %p (%s)\n",
```

Comment:
Perhaps you want to change "Invalidating10" to "Invalidating11" here.

Reply:
done